lib/miq_automation_engine/service_models/miq_ae_service_storage_profile.rb

```diff
@@ -0,0 +1,4 @@
+module MiqAeMethodService
+  class MiqAeServiceStorageProfile < MiqAeServiceModelBase
+  end
```

Comment:
@gmcculloug if we want to be able to get the associated storages do we have to expose the join model (`:storage_profile_storages`) or just `:storages`?